### DISCUSSION TITLE
usage of ag-grid inside BudiBase

### DISCUSSION THREAD

**Discussion by FrancoisLamotte** [Budibase/budibase#3454] [Help] [answered]
Hello

I've discovered BB through a search for ag grid with svelte (as they currently did not support yet Svelte)
=> https://github.com/Budibase/svelte-ag-grid

As I didn't find any reference here on the forum and on the official doc, if someone could tell me 

1) Is it possible to use the sevlte-ag-grid as a component inside BB ?

2) How far can we go as AG Grid could be customized in many ways ? for example creating dynamic pivot table.

3) AG Grid support 2 ways to render graphics (from a data-grid and from their API). Is it possible to manage them in BB ?

Thank you very much.

Regards

**Comment by mjashanks** [comment, marked as answer]
Hi François.

We used to use Ag Grid for our table component but stopped because 
1. We couldn't it to fit with our themes
2. It was a very heavy component to have to bundle with everything.

So, at the moment, no. Someday, we should be able to write our own components for Budibdase. At that time, AG Grid might be an option again. I don't know when this will happen though, as we keep pushing it back.

I agree that it would be great to have as an option, but it's not going to be our default option for a table.

**Comment by FrancoisLamotte** [comment]
Thank you for your answer
